app/src/main/java/com/example/habitsmasher/ui/dashboard/RecyclerTouchListener.java

```diff
@@ -0,0 +1,754 @@
+package com.example.habitsmasher.ui.dashboard;
+
+import android.animation.Animator;
+import android.animation.ObjectAnimator;
+import android.app.Activity;
+import android.graphics.Rect;
+import android.util.DisplayMetrics;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.VelocityTracker;
+import android.view.View;
+import android.view.ViewConfiguration;
+import android.view.animation.DecelerateInterpolator;
+import android.widget.ListView;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Listener class that handles touch interactions with a RecyclerView
+ * Implementation of this class came from this source:
+ * Name: Velmurugan
+ * Date: March 4, 2021
+ * URL: https://howtodoandroid.com/android-recyclerview-swipe-menu
+ */
+public class RecyclerTouchListener implements RecyclerView.OnItemTouchListener, OnActivityTouchListener {
+    private static final String TAG = "RecyclerTouchListener";
+    Activity act;
+
+    /*
+     * independentViews are views on the foreground layer which when clicked, act "independent" from the foreground
+     * ie, they are treated separately from the "row click" action
+     */
+    List<Integer> independentViews;
+
+    /*
+     * Option views are the views on the background layer which acts as the different
+     * clickable options when swipe menu spawned
+     */
+    List<Integer> optionViews;
+
+    // Cached ViewConfiguration and system-wide constant values
+    private int touchSlop;
```

Comment:
Can all these fields be **renamed** to start with an **underscore**, to match the expected style in the other files?